### PR TITLE
Build /iot-management page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build `/internet-of-things/iot-management` page
- Add `IoT Management` on secondary navigation
- Update contact form

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Be sure to test on mobile, tablet and desktop screen sizes
- Please check page against [copy doc](https://docs.google.com/document/d/1C-1ywVzBHkc4I3f-G7Rpy-epaXdbPisbolHUC1Jb7cE/edit#) 
- Please refer to this [brief](https://docs.google.com/document/d/1ENKan1AIRTkzdARyYYE6U9lW1jhw5sRVY76GKrRRa2Y/edit) and [doc](https://docs.google.com/document/d/1Se-fD5hhb7EaR75jd_1D-JkADmg6aLjUs7_bScATvwE/edit#) for contact form 
## Issue / Card

Fixes [#4671](https://github.com/canonical-web-and-design/web-squad/issues/4671) & [#4673](https://github.com/canonical-web-and-design/web-squad/issues/4673)

## Screenshots

IoT Management - Manage Ubuntu Core devices | Ubuntu (2021-12-09 11-56-43)

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @minkyngkm looks good 👍

File: templates/internet-of-things/iot-management.html [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+            ) | safe
+          }}
+          <h3 class="p-heading-icon__title p-heading--4">Whitepaper</h3>
+        </div>
+      </div>
+      <p><a class="p-link--inverted"href="/iot-device-management-whitepaper?_ga=2.90790234.1537229770.1638375570-1154790289.1626446765">Secure IoT device management&nbsp;&rsaquo;</a></p>
```
Comment (sowasred2012) [inline templates/internet-of-things/iot-management.html:100]:
```suggestion
      <p><a class="p-link--inverted"href="/engage/iot-device-management-whitepaper?_ga=2.90790234.1537229770.1638375570-1154790289.1626446765">Secure IoT device management&nbsp;&rsaquo;</a></p>
```

File: templates/internet-of-things/iot-management.html [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+            ) | safe
+          }}
+          <h3 class="p-heading-icon__title p-heading--4">Whitepaper</h3>
+        </div>
+      </div>
+      <p><a class="p-link--inverted p-link--external"href="/engage/smart-start-whitepaper">Accelerating IoT device time to market&nbsp;&rsaquo;</a></p>
```
Comment (sowasred2012) [inline templates/internet-of-things/iot-management.html:206]:
```suggestion
      <p><a class="p-link--inverted" href="/engage/smart-start-whitepaper">Accelerating IoT device time to market&nbsp;&rsaquo;</a></p>
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Looking good - just two small changes. We should ping marketing (David or Lucia) about a form ID too.

File: navigation.yaml [hunk 3]
```diff
@@ -1226,6 +1226,8 @@ iot:
       path: /internet-of-things/networking
     - title: Smart city
       path: /internet-of-things/smart-city
+    - title: IoT Management
+      path: /internet-of-things/iot-management
```
Comment (sowasred2012) [inline navigation.yaml:1230]:
```suggestion
      path: /internet-of-things/management
```

**Review by sowasred2012 — APPROVED** [review]
LGTM - just need that form ID before it gets merged

**Pull request merged**